.circleci/config.yml

```diff
@@ -18,8 +18,11 @@ jobs:
       - run:
           name: 'pylint'
           command: |
+            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox dev_env.sh
```

Comment:
Should we move this into some directory that we can persist for any other step to reuse?

Reply:
Yeah we should, I was being lazy

Reply:
It just becomes a question of whether we think the added complexity is worth it for a file we source twice

Reply:
So for single job workflows, there's no need to generate a tmp workspace. We can access files across runs. I just put the cp in the env step of the job.